### DISCUSSION TITLE
I want alias support QAQ!

### DISCUSSION THREAD

**Discussion by AAAflowerseller** [Future-Scholars/paperlib#394] [Ideas]
Dear Developers, 
Thank you for this elegant work and I love it so much.
But, if there is a editable alias column, I will be a the happiest girl in this world!
Ps. adaptive DPI setting will be better. I really want to tucao that my mentor doesn‘t provide us new computers, so I have to painfully read papers facing my 1080p screen. Although the screen is big, what can 1080p do?
Thank you again. This tool helps me a lot!

**Comment by GeoffreyChen777** [comment]
Thank you.

1. alias for what? publication?
2. could you tell me more about this adaptive DPI? DPI is a fixed measurement of your monitor. Paperlib can do nothing about that.

If you don't know how to describe it in English, just use Chinese, I speak both.

**Reply by AAAflowerseller** [reply]
谢谢你的回复！
alias就是能自定义论文的别名，但不是对论文进行重命名，那样太规矩了。最好能支持图标或者适应输入法自带的图案，我不确定大家是否都有这样的需求，但是论文名字好难记啊，过几个月就忘了，我得用自己的话标注一下qaq，先有的分组、标签还有标记满足不了我这种p人的需求。
dpi这个我不确定对应软件设计里专业的名词是什么，我只是想要控制全局的缩放比例，或者文字大小，我的屏幕太花了看不清，zotero倒是能看清，但我只想用这个满足我审美需求的软件。
![Snipaste_2024-03-11_21-41-41](https://github.com/Future-Scholars/paperlib/assets/116947227/7e9842f0-8a7a-4f7d-89b1-5f55d40afa4b)

**Reply by GeoffreyChen777** [reply]
1. 这个需求有点小众。。 如果你想给论文别名，并显示，你可以在note（笔记）里写上。然后显示该列。你就当做他是alias。
2. 我看你用的，应该是2.2.9版本，这个版本唯一能变大字体的方式，是 ctrl(cmd) + shift + '=', 缩小是 ctrl(cmd) + '-'。3.0 beta 版本已经支持字体大小选择了，你可以试用。

2.2.9 版本不会再继续开发了。除非重大bug。下一个版本就是3.0了。

**Reply by AAAflowerseller** [reply]
谢谢！更新之后一切都舒服起来了！
以及别名真的很小众吗，大家都是怎么记住论文名字的啊。
我用了笔记的功能，很好用！做一句话总结很舒适，不过可以加一个功能，点击笔记栏就可以输入文字，而不用打开属性编辑的页面才能输入笔记吗？

**Reply by GeoffreyChen777** [reply]
你指的笔记栏是哪里，中间的笔记列？还是右侧详情那个面板

**Reply by AAAflowerseller** [reply]
中间的笔记列~以及更新了最新版的后，笔记编辑框里无法打空格了，不知道是不是只有我有这个问题sad

**Reply by GeoffreyChen777** [reply]
你好，已经复现，我大概知道是什么问题。马上修复。

**Reply by GeoffreyChen777** [reply]
已修复，等下更新软件就好。

中间的笔记列 点击就能修改的话，实现起来有点麻烦，暂时不会做。

**Reply by AAAflowerseller** [reply]
谢谢大佬，现在已经很好了！期待后续的更新！

**Reply by GeoffreyChen777** [reply]
Track the feature of alias at #530
